src/TopApp.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import axios from 'axios';
+import { Container } from 'semantic-ui-react';
+import { ToastContainer } from 'react-toastify';
+import App from './App';
+// import PrivateRoute from './PrivateRoute';
+import { UserContext, userContext } from './userContext';
+import 'react-toastify/dist/ReactToastify.css';
+
+axios.defaults.baseURL = 'http://localhost:4000/api';
```

Comment:
Axios configuration should be moved to better place as far as we will use it in most of components.

Reply:
could you please suggest where to move?

Reply:
Maybe you should move it to ```src/config/api.js```
Look at the  [ideas of other people](https://github.com/search?p=2&q=axios.defaults.baseURL&type=Code)